.github/workflows/auto-publish.yml

```diff
@@ -29,3 +29,28 @@ jobs:
           verbose: true
           user: __token__
           password: ${{ secrets.PYPI_API_TOKEN }}
+
+  release-nwbpanel-image:
+    name: NWB Panel Image
+    runs-on: ubuntu-latest
+    needs: publish-pypi
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
Please use a `secrets.GITHUB_USERNAME`, the `github.actor` is whoever originally pushes the workflow, which may not be the same user the `GITHUB_TOKEN` credentials match